test/chunking-form/samples/dynamic-import-chunking/_expected/cjs/main.js

```diff
@@ -11,7 +11,7 @@ function fn (num) {
 }
 
 function dynamic (num) {
-  return Promise.resolve(require("./dep2.js"))
+  return Promise.resolve({ default: require("./dep2.js") })
   .then(dep2 => {
     return dep2.mult(num);
   });
```

Comment:
As you can see here with this change, the code will be broken (you would need to write `dep2.default.mult(num)` to make it work).

Reply:
This is no longer happening now.